core-java-modules/core-java-os/src/test/java/com/baeldung/java9/process/ProcessUnderstandingUnitTest.java

```diff
@@ -83,14 +61,6 @@ public void givenSubProcess_whenDestroyed_thenCheckIfAlive() throws IOException,
         assertFalse(process.isAlive());
     }
 
-    @Test
```

Comment:
We are killing a "random" process. This can work or no, depending on if that process exists or not. Most likely it doesn't exist, so the test fails.